aws/client.py

```diff
@@ -163,6 +166,10 @@ def get(self,
             profiles=None,
             regions=None,
             result_from_error=None):
+
+        if service_name in SERVICES_WITHOUT_REGIONS:
+            regions = ["us-east-1"]
+
```

Comment:
@g-k Thoughts on this?

Reply:
Not great, but we're hardcoding stuff anyway so why not.

Reply:
Haha totally.